policyengine_us/parameters/gov/usda/snap/max_allotment.yaml

```diff
@@ -14,287 +14,343 @@ main:
       2022-10-01: 1_177
+      2023-10-01: 1_253
     4:
       2020-10-01: 1_068
       2021-01-01: 1_228
       2021-10-01: 1_370
       2022-10-01: 1_494
+      2023-10-01: 1_591
     5:
       2020-10-01: 1_268
       2021-01-01: 1_459
       2021-10-01: 1_627
       2022-10-01: 1_774
+      2023-10-01: 1_890
     6:
       2020-10-01: 1_522
       2021-01-01: 1_751
       2021-10-01: 1_952
       2022-10-01: 2_129
+      2023-10-01: 2_268
     7:
       2020-10-01: 1_682
       2021-01-01: 1_935
       2021-10-01: 2_158
       2022-10-01: 2_354
+      2023-10-01: 2_506
     8:
       2020-10-01: 1_923
       2021-01-01: 2_211
       2021-10-01: 2_466
       2022-10-01: 2_690
+      2023-10-01: 2_865
   AK_RURAL_2:
     1:
       2020-10-01: 390
       2021-01-01: 448
       2021-10-01: 500
-      2022-10-01: 545 
+      2022-10-01: 545
+      2023-10-01: 581
     2:
       2020-10-01: 715
       2021-01-01: 822
       2021-10-01: 917
       2022-10-01: 1_000
+      2023-10-01: 1_065
     3:
       2020-10-01: 1_024
       2021-01-01: 1_177
       2021-10-01: 1_313
       2022-10-01: 1_432
+      2023-10-01: 1_525
     4:
       2020-10-01: 1_300
       2021-01-01: 1_495
       2021-10-01: 1_667
       2022-10-01: 1_819
+      2023-10-01: 1_937
     5:
       2020-10-01: 1_544
       2021-01-01: 1_776
       2021-10-01: 1_980
       2022-10-01: 2_160
+      2023-10-01: 2_300
     6:
       2020-10-01: 1_853
       2021-01-01: 2_131
       2021-10-01: 2_376
       2022-10-01: 2_592
+      2023-10-01: 2_760
     7:
       2020-10-01: 2_048
       2021-01-01: 2_355
       2021-10-01: 2_626
       2022-10-01: 2_865
+      2023-10-01: 3_051
     8:
       2020-10-01: 2_340
       2021-01-01: 2_692
       2021-10-01: 3_002
       2022-10-01: 3_274
+      2023-10-01: 3_487
   GU:
     1:
       2020-10-01: 300
       2021-01-01: 345
       2021-10-01: 369
       2022-10-01: 415
+      2023-10-01: 430
     2:
       2020-10-01: 551
       2021-01-01: 634
       2021-10-01: 677
       2022-10-01: 761
+      2023-10-01: 788
     3:
       2020-10-01: 789
       2021-01-01: 908
       2021-10-01: 969
       2022-10-01: 1_090
+      2023-10-01: 1_129
     4:
       2020-10-01: 1_002
       2021-01-01: 1_153
       2021-10-01: 1_231
       2022-10-01: 1_385
+      2023-10-01: 1_434
     5:
       2020-10-01: 1_190
       2021-01-01: 1_369
       2021-10-01: 1_462
       2022-10-01: 1_644
+      2023-10-01: 1_703
     6:
       2020-10-01: 1_428
       2021-01-01: 1_643
       2021-10-01: 1_754
       2022-10-01: 1_973
+      2023-10-01: 2_044
     7:
       2020-10-01: 1_579
       2021-01-01: 1_816
       2021-10-01: 1_939
       2022-10-01: 2_181
+      2023-10-01: 2_259
     8:
       2020-10-01: 1_804
       2021-01-01: 2_075
       2021-10-01: 2_216
       2022-10-01: 2_493
+      2023-10-01: 2_581
   HI:
     1:
       2020-10-01: 375
       2021-01-01: 432
       2021-10-01: 472
       2022-10-01: 538
+      2023-10-01: 527
```

Comment:
Just checking that Hawaii numbers fell? Can you please double check them?

Reply:
The numbers provided are correct, the Hawaii numbers did fall despite[increased price levels](https://www.bls.gov/regions/west/news-release/consumerpriceindex_honolulu.htm):
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/f3fe5d9d-df37-4c71-b747-0436aedd9852)

This is due to the [adjustment](https://www.hawaiinewsnow.com/2023/08/08/despite-rising-prices-hawaii-residents-poised-see-cut-food-stamp-benefits-this-fall/) of Hawaii’s Thrifty Food Plan funding by the [Department of Agriculture ](https://www.fns.usda.gov/cnpp/tfp-akhi)